.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,26 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.owner.login != github.event.pull_request.base.repo.owner.login
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit black codespell flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101,B105,B108,B110,B112,B306,B310,B404,B506,B602,B603,B605,B607 .
+      - run: black --check . || true
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
+      - run: flake8 --ignore=B001,B006,B007,C402,C404,C408,E117,E121,E126,E128,E131,E203,E241,E261,E302,E303,E722,E741,F401,F841,W291,W293,W391,W504,W605
+                    --max-complexity=26 --max-line-length=190 --show-source --statistics .
+      - run: isort --check-only --profile black . || true
+      - run: pip install -r requirements.txt || pip install --editable .  || pip install . || true
```

Comment:
Is the obsolete, right?
```suggestion
```

Reply:
Nope.  If any of the customary files are ever put in place then the Action  automatically takes advantage of them.

Reply:
motionEyeOS is no Python application, so it it will be never build/installed via `pip install` 😉. This is why I suggested to remove the `requirements.txt` since it gives the impression it would be a Python application and required to run `pip install -r requirements.txt` in order to build/install it.

But I've actually never tried to build it so far, if Python is invoked at some point and indeed requires additional modules, there is a point to add a `requirements.txt`.